packages/migrate/Migration.js

```diff
@@ -185,6 +185,14 @@ class Migration {
    * @param  {Object}   options  config and command-line
    */
   async run(options) {
+    // check --migrate-none
+    if (config.migrateNone || config['migrate-none']) {
+        if (config.events) {
```

Comment:
Is this check necessary?  I don't think we check this in other places, do we?  We can assume that `config.events` exists, right?

Reply:
yes, config.events exists, remove the if-statement